tests/CommandUnishTestCase.php

```diff
@@ -377,49 +377,6 @@ public function drush($command, array $args = [], array $options = [], $site_spe
         return $return;
     }
 
-  /**
-   * Override the run method, so we can add in our code coverage data after the
-   * test has run.
-   *
-   * We have to collect all coverage data, merge them and append them as one, to
-   * avoid having phpUnit duplicating the test function as many times as drush
-   * has been invoked.
-   *
-   * Runs the test case and collects the results in a TestResult object.
-   * If no TestResult object is passed a new one will be created.
-   *
-   * @param  \PHPUnit_Framework_TestResult $result
-   * @return \PHPUnit_Framework_TestResult
-   * @throws \PHPUnit_Framework_Exception
-   */
-    public function run(\PHPUnit_Framework_TestResult $result = null)
```

Comment:
This chunk was an OO error on PHPUnit 5. Not sure how to make both 5 and 6 happy with same code. See https://circleci.com/gh/drush-ops/drush/3338

I'm sad to lose this code coverage collecting during $this->drush() calls but we are not using it anyway. If anyone can fix that error we will bring this code back